apps/skilavottord/ws/src/app/modules/recyclingPartner/recyclingPartner.service.ts

```diff
@@ -1,70 +1,63 @@
-import { Inject, Injectable } from '@nestjs/common'
+import { Injectable } from '@nestjs/common'
 import { InjectModel } from '@nestjs/sequelize'
 
-import type { Logger } from '@island.is/logging'
-import { LOGGER_PROVIDER } from '@island.is/logging'
-
 import { RecyclingPartnerModel } from './recyclingPartner.model'
+import {
+  CreateRecyclingPartnerInput,
+  DeleteRecyclingPartnerInput,
+  UpdateRecyclingPartnerInput,
+} from './recyclingPartner.input'
 
 @Injectable()
 export class RecyclingPartnerService {
   constructor(
     @InjectModel(RecyclingPartnerModel)
     private recyclingPartnerModel: typeof RecyclingPartnerModel,
-    @Inject(LOGGER_PROVIDER)
-    private logger: Logger,
   ) {}
 
-  async findByPartnerId(companyId: string): Promise<RecyclingPartnerModel> {
-    this.logger.info(`Finding recyclingPartner by companyId - "${companyId}"`)
-    try {
-      return await this.recyclingPartnerModel.findOne({
-        where: { companyId },
-      })
-    } catch (error) {
-      this.logger.error('error findByPartnerId:' + error)
-    }
+  async findAll(): Promise<RecyclingPartnerModel[]> {
+    return await this.recyclingPartnerModel.findAll()
   }
 
-  async findAll(): Promise<RecyclingPartnerModel[]> {
-    this.logger.info('find all recyclingPartners...')
-    try {
-      const res = await this.recyclingPartnerModel.findAll()
-      this.logger.info(
-        'findAll-recyclingPartners result:' + JSON.stringify(res, null, 2),
-      )
-      return res
-    } catch (error) {
-      this.logger.error('error finding all recyclingPartners:' + error)
-    }
+  async findAllActive(): Promise<RecyclingPartnerModel[]> {
+    return this.recyclingPartnerModel.findAll({
+      where: { active: true },
```

Comment:
Following my previous comment ... let's remove the where clause.

Reply:
take a look at my comment above